src/digisos/skjema/familie/forsorgerplikt/RegistrerteBarn.tsx

```diff
@@ -138,7 +138,6 @@ const RegistrerteBarn = () => {
                                         feil={feil_}
                                         maxLength={3}
                                         required={false}
-                                        noValidate={true /* Unngå at nettleser validerer og evt. fjerner verdien */}
```

Comment:
Gjelder ikke utfordringen i kommentaren lenger?

Reply:
Jeg mener å huske at dette ikke var en gyldig prop på `Input` i nav-frontend, som er grunnen til at den måtte bort. Jeg sjekker. :)

Reply:
@mrbjoern  Fikk du sjekket denne? Jeg husker ikke :)

Reply:
Leste litt på MDN https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form og det ser vel egentlig ut til at `noValidate` kun kan brukes på `<form>`. Eventuelt kan `formNoValidate` brukes på `button` og input av typen `submit` eller `image`.

Jeg sjekket raskt på både min branch og `master`. Begge to har et inputfelt for tall, og hvis du skriver inn noe ugyldig (eks "10e") vil inputfeltet tømmes. Dette gjelder begge branchene. Vi kan jo ta en diskusjon på om vi skal gjøre en bedre / annerledes validering her, men det hører kanskje hjemme i en egen PR. :)